OpenSim/Simulation/SimbodyEngine/Joint.cpp

```diff
@@ -554,7 +554,7 @@ void Joint::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNumber)
             XMLDocument::renameChildNode(aNode, "location", "location_in_child"); 
             XMLDocument::renameChildNode(aNode, "orientation", "orientation_in_child");
         }
-        // Version 30501 converted Connector_Body_ to Connector_PhysicalFrame_
+        // Version 30501 converted Socket_Body_ to Connector_PhysicalFrame_
```

Comment:
I don't think this change is correct (version 30501 couldn't have converted from "Socket_Body_" because sockets didn't exist at that point).